### DISCUSSION TITLE
Codespaces stuck at setting up your codespaces

### DISCUSSION THREAD

**Discussion by end-9214** [community/community#75343] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I have been waiting for almost 10-15 hours and my Codespace is still not available. Its showing "Setting up your Codespaces". I tried creating new Codespace but i m still unsuccessful. i don't know what to do know.

**Comment by adisuyash** [comment]
I have the same question that I'm also stuck at this screen
Hope @github resolves this issue
<img width="338" alt="image" src="https://github.com/community/community/assets/116362593/ef1206d1-d12f-437a-82b4-3df61f2cf989">

**Reply by Azaz22ali** [reply]
Hi Adisuyash,
I hope this message finds you well. I've been encountering a persistent issue with my Codespace for the last three hours. Unfortunately, despite various attempts, I haven't been able to resolve it.

The problem is that the Codespace remains stuck at "**Setting up your codespace**" and doesn't proceed further. I've previously used this Codespace without any issues, but the current situation is hindering my work.

Could you kindly provide guidance on how to overcome this issue? Any assistance you can offer to help me get the Codespace back to normal operation would be greatly appreciated.

**Reply by adisuyash** [reply]
Hi @Azaz22ali, my codespaces have started to run automatically.
I think that was a bug or something...
But what you can do is don't open the codespaces in your browser, instead use Open in VSCode option...
If that doesn't work, clear your browser cache or try using another browser 🙌

**Comment by end-9214** [comment, marked as answer]
i tried changing my internet connection and it worked just fine !! :D

**Comment by Azaz22ali** [comment]
Thank you for your response,I will try and implement of what you suggested.

And will let you know!

**Comment by Akidul-islam** [comment]
I'm also facing same problem. Sometimes It's work properly, sometimes It stuck the screen. How can It resolved🫠

**Reply by end-9214** [reply]
just try changing your internet connection or use a VPN, this should work 100%
